devfile.yaml

```diff
@@ -5,7 +5,10 @@ components:
   - name: tools
     container:
       image: registry.redhat.io/devspaces/udi-rhel8:3.9
-      memoryLimit: 3Gi
+      cpuLimit: '4'
+      cpuRequest: '1'
+      memoryLimit: '3Gi'
+      memoryRequest: '2Gi'
```

Comment:
the order listed is different from other PRs where you have memory first, cpu second.